### PR TITLE
Adding Report Collector docker creation

### PR DESCRIPTION
**Pull request by cberkhoff**
Also adding some utilities to the docker. I can remove them after we're done setting up the sharded environment.

### CODE REVIEW

File: docker/report_collector.Dockerfile [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+RUN set -eux; \
+    cargo build --bin report_collector --release --no-default-features --features "cli test-fixture web-app real-world-infra compact-gate"
+
+# Copy them to the final image
+FROM rust:slim-bookworm
+ENV RC_BIN_PATH=/usr/local/bin/report_collector
+ENV CONF_DIR=/etc/ipa
+ARG SOURCES_DIR
+
+RUN apt-get update && apt-get install -y ca-certificates && rm -rf /var/lib/apt/lists/*
```
Comment (akoshelev) [inline docker/report_collector.Dockerfile:18]: I am not sure if we need certs on RC. Helpers need them to start the web servers

File: docker/report_collector.Dockerfile [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# syntax=docker/dockerfile:1
+ARG SOURCES_DIR=/usr/src/ipa
+FROM rust:bookworm AS builder
+ARG SOURCES_DIR
+
+# Prepare report collector binaries
+WORKDIR "$SOURCES_DIR"
+COPY . .
+RUN set -eux; \
```
Comment (akoshelev) [inline docker/report_collector.Dockerfile:9]: report collector only needs two features "cli" and "test-fixture"

**Review by akoshelev — APPROVED** [review]
we should fix the RC command and be good to go

File: docker/report_collector.Dockerfile (hunk 1 above)
Comment (cberkhoff) [reply docker/report_collector.Dockerfile:18]: Not sure either, I'll keep them

File: docker/report_collector.Dockerfile (hunk 2 above)
Comment (cberkhoff) [reply docker/report_collector.Dockerfile:9]: I checked and it needs all those features

**Pull request merged**